src/nm.c

```diff
@@ -431,9 +431,6 @@ write_tunnel_params(const NetplanNetDefinition* def, GKeyFile *kf)
 static void
 write_dot1x_auth_parameters(const NetplanAuthenticationSettings* auth, GKeyFile *kf)
 {
-    if (auth->eap_method == NETPLAN_AUTH_EAP_NONE)
-        return;
-
```

Comment:
question: this is because this value is the default when the EAP method is unknown, right?

Reply:
exactly.

It is initialized like that (`auth->eap_method = NETPLAN_AUTH_EAP_NONE;`) in types.c:reset_auth_settings()